rest-servlet/src/main/java/org/hawkular/metrics/restServlet/influx/query/validation/AggregationFunction.java

```diff
@@ -44,7 +44,17 @@
     /** */
     MEDIAN("median"),
     /** */
-    PERCENTILE("percentile", new PercentileAggregatorRule());
+    PERCENTILE("percentile", new PercentileAggregatorRule()),
+    /** */
+    TOP("top"),
+    /** */
+    BOTTOM("bottom"),
+    /** */
+    HISTOGRAM("histogram"),
+    /** */
+    MODE("mode"),
+    /** */
+    STDDEV("stddev");
 
```

Comment:
All functions which takes extra arguments (top, bottom, histogram) should have a dedicated validator. See how PERCENTILE is declared.